src/Symfony/Component/HttpKernel/Tests/HttpCache/HttpCacheTest.php

```diff
@@ -172,7 +172,7 @@ public function testRespondsWith304OnlyIfIfNoneMatchAndIfModifiedSinceBothMatch(
         $t = \DateTime::createFromFormat('U', time() - 3600);
         $this->request('GET', '/', ['HTTP_IF_NONE_MATCH' => '12345', 'HTTP_IF_MODIFIED_SINCE' => $t->format(\DATE_RFC2822)]);
         $this->assertHttpKernelIsCalled();
-        $this->assertEquals(200, $this->response->getStatusCode());
+        $this->assertEquals(304, $this->response->getStatusCode());
```

Comment:
The test case name no longer matches what is tested here. I think we should look into it and decide if it is enough to rename it or if we need to rework the test itself too.